src/loading.jl

```diff
@@ -1,22 +1,32 @@
 import ..ClimaComms
 
-export import_required_backends
+export @import_required_backends
 
-function mpi_is_required()
-    return context_type() == :MPICommsContext
-end
+"""
+    mpi_is_required()
 
-function mpi_ext_is_not_loaded()
-    return isnothing(Base.get_extension(ClimaComms, :ClimaCommsMPIExt))
-end
+Returns a Bool indicating if MPI should be loaded, based on the
+`ENV["CLIMACOMMS_CONTEXT"]`. See [`ClimaComms.context`](@ref) for
+more information.
```

Comment:
Yes, and that is why I added `see [`ClimaComms.context`](@ref) for
more information.`. Would you prefer we duplicate these docs?

Reply:
No, my suggestion was the opposite, to just keep "see ClimaComms" and remove the reference to `CLIMACOMMS_CONTEXT`. Or maybe say that "most of the time, it can be controlled by ENV, but see XXX for precise rules".

Reply:
Also, `CLIMACOMMS_CONTEXT` does technically define the context, we just have some fallback logic if it doesn't match.

Reply:
Okay, I don't feel strongly about this. Feel free to phrase it as you see fit (the most important thing is having the reference to context, which is already there)